docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.9"
+
+services:
```

Comment:
I appreciate the initiative but we didn't mention anything about Dockerizing this service now.

Reply:
Alright boss